homework.md

```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 x 3 x 6 x 1 x 10 x 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+To make website to look the way we want
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used both on client-side and server side of the website, making website to behave in certain way and be interactive
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. It saves the versions of the document, every change is dated and it is clear who made change. We have control over versions and can choose which one to use.
+2. It allows for people to collaborate on the same project, changes are tracked.
+3. It marks and shows very clearly, what changes where made, when and why, we do not need to look manually for the difference between versions. Easier to track mistakes
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it to your own account (on GitHub), it looks as if it is yours. You can make your own changes on the forked repository.
 
-## Question 10 
+## Question 10
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You copy repository to your own machine. Any changes you will make, you will contribute to original repository
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+You ask the owner of repository to pull in changes you have made.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It marks and shows what changes, when and by whom were made.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+You send the changes you made on your local machine to the repository to Github browser
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+Github is repository hosting service, where we can use Git version control and collaborate
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Commit basically means "save"
```

Comment:
Also it means that you are creating a historical version of the file you are working on in case you wanted to revert back any changes safely, without losing any data